recipes/array2gif/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
```

Comment:
You might consider using jinja templating for the `package` and `source` sections as shown in the example:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L4-L21